### PR TITLE
Feature 54329 folder component

### PR DESCRIPTION
**Pull request by TheWalkingLeek**

### CODE REVIEW

File: frontend/app/templates/components/folder-show.hbs [hunk 1]
```diff
@@ -1,6 +1,25 @@
...
+  <div id={{concat "folderHeading-" @folder.id}}>
+    <div class="mb-0" role="button" data-toggle="collapse" data-target={{concat "#folderCollapse-" @folder.id}} aria-expanded="true" aria-controls={{concat "#folderCollapse-" @folder.id}}>
+      <div class="d-flex flex-row justify-content-between">
+        <div>
+          <h2 class="folder-header d-inline">{{@folder.name}}</h2>
+        </div>
+        <div></div>
+        <div>
+          <LinkTo @route="folders.edit" @model={{@folder}}>
+            <img class="icon-button folder-edit-icon" src="/ember/assets/images/edit.svg" alt="edit">
```
Comment (lucashabersaat) [inline frontend/app/templates/components/folder-show.hbs:11]: Not sure, but should already use the new icons I assume?

File: frontend/app/templates/components/folder-show.hbs [hunk 2]
```diff
@@ -1,6 +1,25 @@
-{{@folder.name}}
-{{@folder.description}}
+<div class="accordion mt-3" id={{concat "folderAccordion-" @folder.id}}>
+  <div id={{concat "folderHeading-" @folder.id}}>
+    <div class="mb-0" role="button" data-toggle="collapse" data-target={{concat "#folderCollapse-" @folder.id}} aria-expanded="true" aria-controls={{concat "#folderCollapse-" @folder.id}}>
```
Comment (lucashabersaat) [inline frontend/app/templates/components/folder-show.hbs:3]: As in team show, you can use the BsCollapse Component.

File: frontend/app/templates/components/folder-show.hbs [hunk 3]
```diff
@@ -1,6 +1,25 @@
...
+      <div class="d-flex flex-row justify-content-between">
+        <div>
+          <h2 class="folder-header d-inline">{{@folder.name}}</h2>
+        </div>
+        <div></div>
```
Comment (lucashabersaat) [inline frontend/app/templates/components/folder-show.hbs:8]: Remove?

**Comment by TheWalkingLeek** [comment]
Closing this since it's included in #262

**Pull request closed without merging**